system/jlib/jstring.cpp

```diff
@@ -2486,10 +2486,10 @@ StringBuffer & ncnameUnescape(char const * in, StringBuffer & out)
 
 
 
-bool startsWith(const char* src, const char* dst)
+bool startsWith(const char* src, const char* prefix)
 {
-    while (*dst && *dst == *src) { src++; dst++; }
-    return *dst==0;
+    while (*prefix && *prefix == *src) { src++; prefix++; }
+    return *prefix==0;
 }
 
 bool startsWithIgnoreCase(const char* src, const char* dst)
```

Comment:
Why not fix this one if you are going to fix that one...

Reply:
Energy - this was the only one I was calling.  I'll make the same change.

Reply:
I could complain about endsWith too... but I won't